R/factorize.R

```diff
@@ -128,6 +128,17 @@ factorize_secuTrial.data.frame <- function(data, cl, form, items, short_names) {
 
     data[, paste0(name, ".factor")] <- factorize_secuTrial(data[, name], lookup)
     data <- .move_column_after(data, paste0(name, ".factor"), name)
+    # check for conversion of all else warn // this is expected to never happen
+    if (length(which(is.na(data[, paste0(name, ".factor")]))) >
+        length(which(is.na(data[, name]))) &
+        # exclude audit trail
+        # note: this will also exclude forms with a name starting with
+        #       "at" if shortnames is exported
+        # exclude mnpfs variables since 0 is not converted
+        ! (grepl("^mnpfs", name) | grepl("^at", form))) {
+      warning(paste("Unexpected: Not all factors converted for:", name,
```

Comment:
very minor point... I wouldnt include the ":"s so that it reads "Unexpected: Not all factors converted for variable_name in form form_name"
And thinking about it, isnt this message wrong? The variables are the factors, so shouldn't the message be "Unexpected: variable_name not converted to factor in form form_name"?

Reply:
Ah...I see... "Unexpected: Not all levels converted for variable_name in form form_name" would be better... (the variable is the factor, whereas the options the factor can take are levels)